app/controllers/users_controller.rb

```diff
@@ -273,6 +273,7 @@ def user_params
       :message_on_new_phenotype,
       :message_on_phenotype_comment_reply,
       :message_on_snp_comment_reply,
+      user_phenotypes_attributes: [:id, :variation, :_destroy]
```

Comment:
I don't think the `_destroy` needs to be in there, as it's just for rails to figure out the HTTP method. Usually browsers don't support all of them, so Rails uses the additional parameter to work around that constraint.